linux_priv.c:82:90: error: invalid application of 'sizeof' to incomplete type 'struct termios'
   82 |     rc |= seccomp_rule_add(ctx, SCMP_ACT_ALLOW, SCMP_SYS(ioctl), 1, SCMP_A1(SCMP_CMP_EQ, TCGETS));

--- linux_priv.c
+++ linux_priv.c
@@ -1,5 +1,6 @@
 #include "config.h"
 #include <seccomp.h>
+#include <termios.h>
 #include <errno.h>
 #include <stdlib.h>
 #include <sys/ioctl.h>
